dashboard/app/models/script.rb

```diff
@@ -1451,7 +1451,10 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
 
     # Filter out stages that have a visible_after date in the future
     filtered_lessons = lessons.select {|lesson| lesson.published?(user)}
-    summary[:stages] = filtered_lessons.map {|lesson| lesson.summarize(include_bonus_levels)} if include_lessons
+    if include_lessons
+      summary[:lessons] = filtered_lessons.map {|lesson| lesson.summarize(include_bonus_levels)}
+      summary[:stages] = summary[:lessons] # TODO: remove after corresponding js code change is deployed and not cached anymore
```

Comment:
@uponthesun it looks like this PR was merged 6 days ago so it should have been live in production for at least a few days now. Would you be willing to queue up a PR to remove this line? the script summary is 200KB in some cases even without this duplication, so I'm worried this is slowing down script page load time.